WikiTemplate/iOS/wikitemplate-ios.md

```diff
@@ -59,6 +59,7 @@
 ## Password Managers
 
 - [ ] Verify tapping on 1Password on the slide-out keyboard launches 1Password app and you're able to select the stored credentials
+- [ ] Verify tapping on iCloud Keychain on the slide-out keyboard launches keychain app and auto-fills the stored data
 - [ ] Verify tapping on Bitwarden password manager in the autofill field launches the app and auto-fills the stored data
```

Comment:
Maybe we can get rid of this as well. This was initially added as we had some compatibility issues and don't think I have seen any complains so far. Feel free to get rid of this one as well

Reply:
Ok sure, we can omit it. Thanks